Cargo.toml

```diff
@@ -14,13 +14,17 @@ readme = "README.md"
 travis-ci = { repository = "dtolnay/thiserror" }
 
 [dependencies]
-thiserror-impl = { version = "=1.0.11", path = "impl" }
+thiserror-impl = { version = "=1.0.11", path = "impl", default-features = false }
 
 [dev-dependencies]
 anyhow = "1.0"
 ref-cast = "1.0"
 rustversion = "1.0"
 trybuild = { version = "1.0.19", features = ["diff"] }
 
+[features]
+default = ["std"]
```

Comment:
Note, this is technically a breaking change if anyone was already using `thiserror` with `default-features = false`, as they will now have reduced functionality.

I got burned by this when I added `no_std` to `num` crates -- https://github.com/rust-num/num/issues/297. I've seen a lot of crates brush that off though, whether deliberate or not.